### ISSUE TITLE
Unrecognized version that doesn't follow semantic version

### ISSUE THREAD

**Issue by henryw14** [quaertym/ember-cli-dependency-checker#40]
I just installed <a href="https://github.com/malsup/blockui/">Block UI</a> via bower. There version it specifies in its `bower.json` is `2.70` without a patch version. This causes the cli to tell me: 
```
Invalid Version: 2.70
TypeError: Invalid Version: 2.70
    at new SemVer (/Users/will.henry/Desktop/unikitty/core/smallbiz/node_modules/ember-cli-dependency-checker/node_modules/semver/semver.js:295:11)
```
 If I manually change it to `2.70.0` all is well. But I don't want to do that. How do I ignore this check for `blockUI`? The "whitelist" issue only checked if a dependency didn't have a `bower.json`?

**Comment by quaertym** [comment]
In your `bower.json` you can specify version as '*' but it is best to fix it upstream.

**Comment by henryw14** [comment]
was that fix added post `"ember-cli-dependency-checker": "0.0.7",`?

**Comment by quaertym** [comment]
It is available since `0.0.8`.
